SR2019/2019-10-16-london-kickstart-deny.md

```diff
@@ -0,0 +1,14 @@
+---
+to: Student Robotics 2019, London Segment
```

Comment:
Surely this one should go to the Southampton Segment? Or am I misunderstanding the terminology here?

Reply:
Not quite. The 2 emails need to go to different subsets of the London segment, depending on if we concluded they go to the London event or not.